pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -90,6 +120,29 @@
      */
     CompletableFuture<List<String>> getListAsync(String namespace);
 
+    /**
+     * Get the list of topics under a namespace asynchronously.
+     * <p/>
+     * Response example:
+     *
+     * <pre>
+     * <code>["topic://my-tenant/my-namespace/topic-1",
+     *  "topic://my-tenant/my-namespace/topic-2"]</code>
+     * </pre>
+     *
+     * @param namespace
+     *            Namespace name
+     *
+     * @param mode
+     *            Get topic mode
+     *            use {@link GetTopicsMode#PERSISTENT} to get persistent topics
+     *            use {@link GetTopicsMode#NON_PERSISTENT} to get non-persistent topics
```

Comment:
The non-persistent topic will always return empty, so this interface cannot get the non-Persistent topic

Reply:
Why non-persistent topics will always return empty? I'm not getting the key point here, currently the list topics operation combine the persistent topics and non-persistent topic, I also add test for cover this case.

Reply:
In the client, if it is a non-persistent topic, the request will complete directly
https://github.com/apache/pulsar/blob/d17eb071e663780e5ba3fd6475d088193d8c6154/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/TopicsImpl.java#L168